lib/presenter/controllers/convertion_validation/convert_validation_notifier.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:crypto_app/domain/entities/wallet_crypto_entity.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class ConvertValidationNotifier extends StateNotifier<bool> {
+  ConvertValidationNotifier() : super(false);
+
+  void validateconversion(
+    double convertQuantity,
+    WalletCryptoEntity convertedCurrency,
+    WalletCryptoEntity toConvertCurrency,
+  ) {
+    if (convertQuantity >= convertedCurrency.quantity) {
+      state = false;
+    } else if (toConvertCurrency.crypto.id == '') {
+      state = false;
+    } else if (toConvertCurrency.crypto.id == convertedCurrency.crypto.id) {
+      state = false;
+    } else if (convertQuantity <= 0) {
+      state = false;
```

Comment:
daria pra colocar toda logica num if so, e seria mais facil fazer algo como
```suggestion
final contidade = convertQuantity >= convertedCurrency.quantity;
final idNull = toConvertCurrency.crypto.id == ''
    if (contidade || idNull) {
      state = false;
    } 
```